### PR TITLE
Tests for the graphql-api package

### PR DESCRIPTION
**Pull request by pradel**

### CODE REVIEW

File: packages/graphql-api/__tests__/modules/accounts/index.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+};
+
+describe('AccountsModule', () => {
+  describe('withSchemaDefinition', () => {
+    it('should export typeDefs with schema definition', () => {
+      const accountsGraphQL = AccountsModule.forRoot({
+        accountsServer: accountsServer as any,
+        withSchemaDefinition: true,
+      });
+      expect(accountsGraphQL.typeDefs).toMatch(/schema/);
```
Comment (ardatan) [inline packages/graphql-api/__tests__/modules/accounts/index.ts]: Could you use `print(accountsGraphQL.typeDefs)` here?

Comment (pradel) [reply packages/graphql-api/__tests__/modules/accounts/index.ts]: 👍

**Comment by pradel** [comment]
@ardatan I updated the pr with the last version which fixed the `extendTypeDefs` option but `withSchemaDefinition` option is still not working unfortunately

**Comment by ardatan** [comment]
Could you try it with latest version 0.2.16?

**Comment by pradel** [comment]
Looks like it's still failing

**Comment by ardatan** [comment]
Oh sorry, it is my fault. I forgot to include the changes in latest build. So, it should be fixed in 0.2.17.

**Comment by pradel** [comment]
@ardatan thanks it worked 🎉

**Pull request merged**